src/neighbor_types/neighbor_verlet.h

```diff
@@ -50,8 +51,10 @@ class NeighborVerlet : public Neighbor<t_System>
         system->slice_x();
         auto x = system->x;
 
-        list =
-            t_neigh_list( x, 0, N_local, neigh_cut, 1.0, grid_min, grid_max );
+        list = t_neigh_list( x, 0, N_local, neigh_cut, 1.0, grid_min, grid_max,
+                             max_neigh );
+        max_neigh =
```

Comment:
OK so here you are basically pre-allocating based on some guess and will increase as need to avoid counting every time. We need to document that `max_neigh` parameter in the `VerletList` doxygen - I had to read through the code a bit to figure out what it does.

Reply:
Right, I'll put in a PR to cover that